config/crd/bases/store.externalsecret-operator.container-solutions.com_secretstores.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.3.0
+    controller-gen.kubebuilder.io/version: v0.2.5
```

Comment:
Why are we changing this version here?

Reply:
I think I did a `make manifests` and it updated the CRDs. I'll revert this, one sec.